servers/cromwell/jobs/__main__.py

```diff
@@ -31,13 +33,42 @@
     args, _ = parser.parse_known_args()
 
 app = connexion.App(__name__, specification_dir='./swagger/', swagger_ui=False)
+DEFAULT_CROMWELL_CREDENTIALS = {"cromwell_user": "", "cromwell_password": ""}
+LOADER_ERR_MSG = "Failed to load config.json, using the default config: {0}".format(
+    DEFAULT_CROMWELL_CREDENTIALS)
+
+# Load credentials for cromwell
 config_path = os.environ.get('CROMWELL_CREDENTIALS')
-with open(config_path) as f:
-    config = json.load(f)
+
+# Check if the credentials are provided properly
+try:
+    with open(config_path) as f:
+        config = json.load(f)
+except (IOError, TypeError):
+    print(LOADER_ERR_MSG)
```

Comment:
Nit maybe add a bit more context:
```
print 'Failed to load config: {}'.format(LOADER_ERR_MSG)
```

Reply:
Also, any reason to print this error rather than raise an exception and stop execution?

Reply:
Sorry for the confusion here, I should have call it `LOADER_WARN_MSG`. The reason is that different Cromwell instances are using different authentication. Our team is using HTTP basic auth, which explicitly requires the `config.json`(credentials). But there are some other Cromwell instances using different proxies, for example, no basic auth required, but can only be accessible within a certain intranet. For those cases, config is not need to be provided by the user/developer, and stoping execution may confuse the users.

Reply:
Will change it to `LOADER_WARN_MSG` to eliminate the confusion.

Reply:
Gotcha, thanks for the explanation that sounds good!